src/plots/plotGroup.ts

```diff
@@ -0,0 +1,33 @@
+module Plottable.Plots {
+  export class PlotGroup extends Components.Group {
+    public entityNearest(point: Point): PlotEntity {
+      let closestPlotEntity: PlotEntity;
+      let minDistSquared = Infinity;
+      this.components().forEach((plot: Plot) => {
+        let candidatePlotEntity = plot.entityNearest(point);
+        if (candidatePlotEntity == null) {
+            return;
+        }
+
+        let position = candidatePlotEntity.position;
+        let distSquared = Math.pow(position.x - point.x, 2) + Math.pow(position.y - point.y, 2);
+        if (distSquared <= minDistSquared) {
+            minDistSquared = distSquared;
+            closestPlotEntity = candidatePlotEntity;
+        }
+      });
+
+      return closestPlotEntity;
+    }
+    /**
+     * Adds a Plot to this Plot Group.
+     * The added Plot will be rendered above Plots already in the Group.
+     */
+    public append(component: Component): any {
```

Comment:
Shouldn't this take in a `Plot` and not a `Component`, then?
It should also return `this` and not `any`.